### ISSUE TITLE
Saving Columns in Climatic > Prepare > Specific Events 

### ISSUE THREAD

**Issue by rdstern** [IDEMSInternational/R-Instat#2809]
The three dialogues don't give an option of the column name.  More dangerously, they overwrite without warning.

**Comment by lilyclements** [comment]
Just sorted this for Water Balance. I will now do the same for Start of Rains and Spells

**Comment by volloholic** [comment]
@lilyclements  Status?

**Comment by lilyclements** [comment]
This issue is still standing, but I forgot about this until now.

The problem here is something I'm struggling to explain so I will by the example of Start of Rains (However, it occurs elsewhere, such as Spell Length).

You open a data frame, "dataname" and create a new column called "start_of_rains1"
You press OK and it opens a new data frame ("dataname_by_Year") and that has in it your new column ("start_of_rains1")
If you then reopen the dialog to create a new start of rains column, it is called "start_of_rains1" by default. This is because it is "thinking" the current data frame doesn't have a column called start_of_rains1. Which is true, the current data frame doesn't have a column called start_of_rains1. But the data frame which the new column will go into does have a column called start_of_rains1.
It then overwrites that column.

**Comment by dannyparsons** [comment]
Yes so it's looking in the wrong data frame for determining the suffix number. Would need changes to save control and R method to change this. Can be 0.9 issue.

**Comment by lloyddewit** [comment]
Overwriting without warning is also mentioned in Issue #3543.
